google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -85,22 +96,25 @@ public String apply(Identity identity) {
 
     @Override
     protected Policy fromPb(com.google.iam.v1.Policy policyPb) {
-      Map<Role, Set<Identity>> bindings = new HashMap<>();
+      List<Binding> bindingsList = new ArrayList<Binding>();
```

Comment:
Do we want to allow users to modify the binding in-place?

Reply:
I'd need to allow mutations outside of builders.

Reply:
This could use `ImmutableList.Builder` if we want to keep it immutable